deploy/crds/compute-node.openstack.org_computenodeopenstacks_crd.yaml

```diff
@@ -63,6 +63,22 @@ spec:
             k8sServiceIp:
               description: Kubernetes service cluster IP
               type: string
+            nodesToDelete:
+              description: Nodes to delete upon scale down
+              items:
+                description: NodeToDelete defines the name of the node to delete and
+                  if automatic drain is needed
+                properties:
+                  drain:
+                    description: Automatic draining of the node
```

Comment:
with draining you mean migrate instances off the compute node, or all tasks run on a scale down?

Reply:
I mean do we want to skip all tasks when draining is disable or just the auto migration?

Reply:
auto-migration only!

Reply:
as discussed this is a switch to enable/disable libe migration